main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -51,6 +51,9 @@ class Flix {
     */
   private val internals = List(
     "Prelude.flix" -> StreamOps.readAll(LocalResource.Library.Prelude),
+    "Option.flix" -> StreamOps.readAll(LocalResource.Library.Option),
```

Comment:
Here's an idea: automatically include libraries unless there's a flag, e.g. -Xnostdlib. Tests that need libraries can manually include them.

Actually, this is also useful for development/debugging, when you want a minimal working example.